charts/kubernetes/values.yaml

```diff
@@ -4,6 +4,7 @@ labels: {}
 topology:
   name: cluster
```

Comment:
I think the clusterName and prometheus.url should be top level values, and we should add an icon